src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -46,7 +47,13 @@ const SignInPageContent = (props) => {
                     !props.isSmallScreenWidth && styles.signInPageLeftContainerWide,
                 ]}
             >
-                <View style={[styles.flex1, styles.alignSelfCenter, styles.signInPageWelcomeFormContainer]}>
+                <KeyboardAvoidingView
+                    behavior="padding"
```

Comment:
The linter is complaining about the indentation on this line... but the linter passes locally OK, and this definitely looks like the correct indentation to me.

```
/home/runner/work/App/App/src/pages/signin/SignInPageLayout/SignInPageContent.js
Error:   51:21  error  Expected indentation of 16 space characters but found 20  react/jsx-indent
```

@jasperhuangg can you spot the problem by any chance?